src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -2,11 +2,15 @@
   "dependencies": {
     "System.Runtime": "4.1.1-beta-24403-05",
     "Microsoft.NETCore.Platforms": "1.0.2-beta-24403-05",
-    "System.Console": "4.0.1-beta-24403-05",
+    "System.Console": "4.0.0",
```

Comment:
I think it should be OK to reference the 4.0.0 version of Console if that is all that is needed by this project.